src/axom/quest/DistributedClosestPoint.hpp

```diff
@@ -686,9 +716,12 @@ class DistributedClosestPointImpl
         int nextDst = (m_rank + 1) % m_nranks;
         get_bounding_box_from_conduit_node(queryPartitionBb,
                                            xferNodePtr->fetch_existing("aabb"));
+
         while(xferNodePtr && nextDst != m_rank)
         {
-          double sqDist =
+          const bool isValidDistance = queryPartitionBb.isValid() &&
+            m_objectPartitionBbs[nextDst].isValid();
```

Comment:
Since this explicitly checks for invalid bounding boxes, it is independent of the decision regarding the return value of `squared_distance(bbox,bbox)` for invalid bounding boxes (i.e. `nan` vs. max double).

Reply:
That makes it clear and robust.  Thanks @kennyweiss